### ISSUE TITLE
How do I finetune with validation on my own dataset?

### ISSUE THREAD

**Issue by tannerhs** [Lightning-AI/litgpt#2010]
In the docs, instructions are given for how to make a single json file to feed data for finetuning:

https://github.com/Lightning-AI/litgpt/blob/main/tutorials/finetune_lora.md

How would I use a validation set in addition to my finetuning set for my custom dataset?
